dts/arm/st/stm32f1.dtsi

```diff
@@ -99,6 +99,16 @@
 			status = "disabled";
 			label= "I2C_2";
 		};
+
+		spi1: spi@40013000 {
```

Comment:
Where are SPI2 (40003800) and SPI3 (40003c00)? Other SPI peripherals present on some SoCs in other families appear missing as well.

It looks like you might be putting just the "least common denominator" in the stm32fX.dtsi DTSI files, and filling in additional peripherals in stm32fXYZ.dtsi for the SoC. If that's right, I guess there's more follow-up work to do to add the missing SPI nodes for other SoCs, right?

Reply:
Yes, I added the "least common denominator" in the stm32fX.dtsi and filled the additional SPI nodes in stm32fXYZ.dtsi only to the SOCs where boards use the additional SPI. We could add the rest SPI nodes in stm32fXYZ.dtsi when we add SPI support to other boards.

Reply:
If this is the case (complete set of SPI instances to be provided in a later commit when required) please indicate the scope of current commit in the description.

Reply:
Done.